ert_gui/tools/plot/plot_api.py

```diff
@@ -37,19 +52,20 @@ def _metadata(self, key):
             meta["data_origin"] = "Gen KW"
         return meta
 
-    def get_all_cases_not_running(self):
+    def get_all_cases_not_running(self) -> List:
```

Comment:
Maybe we can either change the name of this function or update the API to match the previous functionality. As it is now it does not return all cases not running. Returns all cases.